masterlist.yaml

```diff
@@ -553,15 +558,16 @@ plugins:
       - <<: *dirtyPlugin
         crc: 0x7f464083
         udr: 1
-      # Version 1.9.32.0 of Update.esm
+      ## (TES5EDIT v3.2.1) Version 1.9.32.0 of Update.esm
```

Comment:
Why not put TES5Edit versions in `util:` like you've done for other entries?

Reply:
On making another change I realized I could have just used. I'll update those when I'm adding clean plug-in info for USLEEP.
```
        util: '[TES5Edit  v3.2.1](http://www.nexusmods.com/skyrim/mods/25859)'
        info: 'A cleaning guide is available [here](http://www.creationkit.com/index.php?title=TES5Edit_Cleaning_Guide_-_TES5Edit).'
```